app/containers/List/ListItem.tsx

```diff
@@ -89,20 +100,21 @@ const Content = React.memo(
 		showActionIndicator = false,
 		theme,
 		heightContainer,
+		rightContainerStyle = {},
 		styleTitle,
 		additionalAcessibilityLabel,
 		additionalAcessibilityLabelCheck,
 		accessibilityRole,
 		accessibilityLabel
 	}: IListItemContent) => {
-		const { fontScale } = useDimensions();
+		const { fontScale } = useWindowDimensions();
```

Comment:
When should we use this directly when we should be using the new responsive fontScale hook?
It doesn't look very clear.

Reply:
I still have this question.